docs/extras/use_cases/extraction/index.mdx

```diff
@@ -12,13 +14,106 @@ Examples of this include:
+
+```python
+schema = {
+    "properties": {
+        "name": {"type": "string"},
+        "height": {"type": "integer"},
+        "hair_color": {"type": "string"},
+    },
+    "required": ["name", "height"],
+}
+```
+
+Let's try it out! To create our extraction chain, we will use the `create_extraction_chain` function.
+
+```python
+inp = """
+Alex is 5 feet tall. Claudia is 1 feet taller Alex and jumps higher than him. Claudia is a brunette and Alex is blonde.
+"""
+```
+
+```python
+chain = create_extraction_chain(schema, llm)
+```
+
+As we can see, we extracted the required entities and their properties in the required format.
+
+```python
+chain.run(inp)
+```
+
+    [{'name': 'Alex', 'height': 5, 'hair_color': 'blonde'},
+     {'name': 'Claudia', 'height': 6, 'hair_color': 'brunette'}]
+
+## Example #2: using a Pydantic schema
+
+We can also use a Pydantic schema instead of a json schema. For this we will use the `create_extraction_chain_pydantic` function.
+
+```python
+from typing import Optional, List
+from pydantic import BaseModel, Field
+from langchain.chains import create_extraction_chain_pydantic
+```
+
+As before we will define our schema, only this time using a Pydantic class.
+
+```python
+class Properties(BaseModel):
+    person_name: str
+    person_height: int
+    person_hair_color: str
+    dog_breed: Optional[str]
+    dog_name: Optional[str]
+```
+
+```python
+chain = create_extraction_chain_pydantic(pydantic_schema=Properties, llm=llm)
+```
+
+```python
+inp = """
+Alex is 5 feet tall. Claudia is 1 feet taller Alex and jumps higher than him. Claudia is a brunette and Alex is blonde.
+Alex's dog Frosty is a labrador and likes to play hide and seek.
+"""
+```
+
+And our output is a list of `Properties` objects (effectively respecting the schema we defined earlier).
+
+```python
+chain.run(inp)
+```
+
+    [Properties(person_name='Alex', person_height=5, person_hair_color='blonde', dog_breed='labrador', dog_name='Frosty'),
+     Properties(person_name='Claudia', person_height=6, person_hair_color='brunette', dog_breed=None, dog_name=None)]
+
+## Going deeper
 
-While normal output parsers are good enough for basic structuring of response data,
-when doing extraction you often want to extract more complicated or nested structures.
-For a deep dive on extraction, we recommend checking out [`kor`](https://eyurtsev.github.io/kor/),
-a library that uses the existing LangChain chain and OutputParser abstractions
-but deep dives on allowing extraction of more complicated schemas.
+As we have seen, by leveraging OpenAI `-0613` models we can extract structured data from unstructured documents with minimal hallucination. For more detail on how to use these chains and different tips and tricks you can use, please check out the [docs](docs/extras/modules/chains/additional/extraction.ipynb) for the extraction chain.
```

Comment:
Maybe we add a final section on `Parsing`

1/ Include the [JSON example](https://python.langchain.com/docs/modules/model_io/output_parsers/pydantic).

2/ Going Deeper

* Link to broader set of [parser docs](https://python.langchain.com/docs/modules/model_io/output_parsers/)